deploy/crds/kubevirt_v1_templatevalidator_cr.yaml

```diff
@@ -5,3 +5,4 @@ metadata:
   namespace: kubevirt
 spec:
   version: v0.6.2
+  template_validator_replicas: 2
```

Comment:
we need to settle for either snake_case or camelCase. Let's just stick to whatever kubernetes prefers, AFAIK camelCase